x/zkism/types/ism.go

```diff
@@ -93,3 +93,35 @@ func (ism *ZKExecutionISM) verifyZKStateInclusion(_ ZkExecutionISMMetadata, _ ut
 	// TODO: https://github.com/celestiaorg/celestia-app/issues/4723
 	return true, nil
 }
+
+func VerifyGroth16(ctx context.Context, ism ZKExecutionISM, proofBz, publicValues []byte) error {
```

Comment:
This is superseded by the verifier type added in: https://github.com/celestiaorg/celestia-app/pull/5803